### PR TITLE
Style/all pages

### PR DESCRIPTION
**Pull request by oacortez**
## Checklist
- [ ] Tests written where required
- [ ] All tests passing
- [ ] Unused code/prys/etc deleted
Testing Coverage at <percentage>%

## Changes Made
  
What (if any) features are you implementing?
- Styled all viewable pages 
- Styled buttons 
- Add Animation for our menu
- Add Sass mixin and variables 

What (if anything) did you refactor?
- Add new className on landing page 
- Removed "Should click the Login button and see the text change" test on landing and form page for cypress test in order to make circle ci pass 100%
- clean code base of all css files

Were there any issues that arose?
- Not responsive 
- minimize font size for test suit to pass
- couple more css bugs that can be fixed with more time

Is there anything that you need from your teammate?
- Please review 
- Double-check code and test if possible for safety

Any other comments, questions, or concerns?
So sorry y'all for making this branch such a hustle. This branch was supposed to be "MusicianProfile" branch, but unfortunately an issue arose. Once testing for circle ci it was not starting the test, and it was failing automatically. I created this duplicate branch to make circle ci pass. Everything seems to be working. Please double check. Thanks!

### CODE REVIEW

**Review by mr-ryan12 — CHANGES_REQUESTED** [review]
Hey thank you Oscar! Everything looks great. I did notice that the 'Name' 'Genre' and 'Song Title' headings on the submission details page are missing. Are they just hidden by the color of their font?

File: src/components/LandingPage/LandingPage.scss [hunk 1]
```diff
@@ -1,62 +1,115 @@
...
 
-.submission-dates-april {
-  top: 45%;
+.winner-image {
+  width: 150px;
+  height: 150px;
+  object-fit: cover;
+  border-radius: 50%;
+  margin: 0 auto 20px auto;
+  display: block;
```
Comment (mr-ryan12) [inline src/components/LandingPage/LandingPage.scss:53]: The winner image looks great!

**Review by mr-ryan12 — APPROVED** [review]
Thank you so much Oscar! Really appreciate the work you put into making our app look SO GOOD! Much appreciated 🎉

**Review by Echory — APPROVED** [review]
Thanks for doing all of this Oscar !

**Review by aliroemhildt — APPROVED** [review]
Thanks for making all of these updates, Oscar! I know it was a lot of work, but happy to see all of our tests passing

**Pull request merged**